server.js

```diff
@@ -219,3 +321,23 @@ function Movies(movies) {
   this.image_url = movies.poster_path;
```

Comment:
concat 'https://image.tmdb.org/t/p/w200_and_h300_bestv2'  to the path.

Reply:
Also add this.tableName = 'movies'; to the Movies constructor and this.tableName = 'yelps' to the Yelp constructor

This is to add the tableName to every instance.